### PR TITLE
Model Json tweak

### PR DESCRIPTION
**Pull request by zNightlord**
The 1.008 dimension issue as mentioned in #429  #451 #452 
Change the behavior of model spawner spawning in just one vertex, this should be a warning of invalid json file instead.

### CODE REVIEW

**Comment by zNightlord** [comment]
a fix for #437

File: MCprep_addon/spawner/mcmodel.py [hunk 1]
```diff
@@ -70,6 +70,16 @@ def rotate_around(
 		(new_pos[1] - offset[1]) * scale[1]
 	))
 
+def getPlanarAxis(verts: List[VectorType]) -> int:
+	"""Return the axis of the face"""
```
Comment (StandingPadAnimations) [inline MCprep_addon/spawner/mcmodel.py:74]:
```suggestion
"""
Given a set of verticies, return the axis of the face

Returns:
    An integer representing the axis
"""
```

**Review by StandingPadAnimations — CHANGES_REQUESTED** [review]
I know this is still a WIP, but I've reviewed the code and commented the things that need to be looked at. Most of them deal with comments and error messages.

Comments shouldn't be vague. Phrases such as "For some reason" don't help other developers understand why something is the way it is. Comments should be clear, not vague.

**Comment by zNightlord** [comment]
I just put random things as notes for myself. 
The "west" face gets appended twice, how matter I check the loop it just appends the last one to the first index so. Have no idea what cause it.
The cake is a json file issue, they just don't add UV coord inside the file.
The face planar check is still WIP for optimizing the model and reuse material is still not there yet.
For model like bed and banner ... in #267 using meshswap instead is there, but it is a bit messy. 
And also adding those models into meshswap blend file

**Comment by TheDuckCow** [comment]
Just noticing this is in draft @zNightlord - please flip to "ready for review" if and when you'd like me to live test!

**Comment by TheDuckCow** [comment]
Oops, sorry @zNightlord  for some reason merging the milestone branch didn't just promote this PR to be based on dev, it closed :/ and it won't let me reopen. You'll need to re-open a new PR just targeting `dev` this time. Sorry about that!

**Pull request closed without merging**